cpu/stm32/periph/gpio_all.c

```diff
@@ -301,10 +357,21 @@ void isr_exti(void)
     }
 #else
     /* only generate interrupts against lines which have their IMR set */
+#if defined(CPU_FAM_STM32MP1)
+    /* TODO: handle if falling or rising */
+    uint32_t pending_isr = (EXTI_REG_RPR & EXTI_REG_IMR);
+    pending_isr |= (EXTI_REG_FPR & EXTI_REG_IMR);
+#else
```

Comment:
Looks like this could be handled the same as it is on stm32g0

Reply:
Indeed, thank you.